### ISSUE TITLE
Logitech G9 and G9x [Call of Duty MW3 Edition] hwdb data

### ISSUE THREAD

**Issue by NoXPhasma** [systemd/systemd#1284]
I've generated hwdb data for the Logitech G9 and the G9x [Call of Duty MW3 Edition], I hope it's okay to add it as Issue, instead of a pull request. I think it would be a bit overdose to fork systemd just to add this few lines :)

```
# Logitech G9x [Call of Duty MW3 Edition]
mouse:usb:v046dpc249:name:Logitech G9x Laser Mouse:
 MOUSE_DPI=400@1000 600@1000 800@1000 1000@1000 1200@1000 1400@1000 1600@1000 1800@1000 2000@1000 2200@1000 2400@1000

# Logitech G9
mouse:usb:v046dpc048:name:Logitech G9 Laser Mouse:
 MOUSE_DPI=400@1000 600@1000 800@1000 1000@1000 1200@1000 1400@1000 1600@1000 1800@1000 2000@1000 2200@1000 2400@1000
```

**Comment by poettering** [comment]
@martinpitt something for you?

**Comment by poettering** [comment]
@NoXPhasma additions like yours are generally preferred as PR btw.

**Comment by NoXPhasma** [comment]
@poettering Ok, I've send a PR (#1293) and will close this issue.

**Comment by poettering** [comment]
@NoXPhasma usually we leave the PR open but add a "Closes #1284" to the PR. github recognizes that and will then also close the indicated issue when the PR is merged.

But anyway, it's also OK if the issue is already closed now...
